src/System.Net.Security/src/System/Net/Security/TlsCipherSuiteData.Lookup.tt

```diff
@@ -0,0 +1,45 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ import namespace="System" #>
+<#@ assembly name="System.Core" #>
+<#@ assembly name="System.Net.Primitives" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ output extension=".cs" #>
+<#@ include file="TlsCipherSuiteNameParser.ttinclude" #><#@
+    include file="..\..\..\..\ref\TlsCipherSuite.cs" #>
```

Comment:
This seems out of date now

Reply:
haven't had a chance to run this on Windows yet - will fix it